scene/gui/file_dialog.cpp

```diff
@@ -40,6 +40,10 @@ FileDialog::GetIconFunc FileDialog::get_large_icon_func = nullptr;
 FileDialog::RegisterFunc FileDialog::register_func = nullptr;
 FileDialog::RegisterFunc FileDialog::unregister_func = nullptr;
 
+void FileDialog::popup_file_dialog() {
```

Comment:
Wouldn't it be better to have a virtual method that each dialog can implement to popup as they see best fit?